webrtc.html

```diff
@@ -3472,6 +3485,23 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
               framerate, <code>degradationPreference</code> indicates what should be prefered.
             </p>
           </dd>
+
+          <dt>DOMString rid</dt>
+          <dd>
+            <p> If set, this RTP encoding will be sent with the RID header extension as
+              defined by [[!RTCWEB-JSEP]].
```

Comment:
Several people (including Adam and Byron, if I recall correctly) requested a way that applications could give some meaning to each layer beyond just a number.  This  is one reason why a string was chosen for the RID instead of an int.  For example, this allows the application to choose "F" for full, "H" for half and "Q" for quarter, or "S", "M", "L", or whatever.  

It's possible that a similar thing could be accomplished by putting the encodings in the right order in the SDP, but that hasn't been fully fleshed out.  It's also possible that the RID can be chosen by the browser and then additional semantics can be signaled to the MCU outside of SDP, which was my original proposal.  

I'm not worried about collisions.  If the application chooses the same RID for two or more encodings, we just throw an exception (I could add some text to make that explicit).  And we could also add some text about keeping the ID short but I'd hope that someone doing something as advanced as simulcast would know that they shouldn't choose a long value for something that's going to go into a header extension.